website/src/main/resources/public/index.html

```diff
@@ -74,7 +74,7 @@ <h1><span class="image right"><img style="width: 50%" src="images/logo_frame.png
                 Lots of friendly volunteers to help you learn Java or chat about everyday
                 things.</small></p>
             <ul>
-                <li>20.000 members</li>
+                <li>24.000 members</li>
```

Comment:
please write 25k or leave it at 20k, thanks.
its too specific otherwise, looks odd

Reply:
Did this